docker-compose.yaml

```diff
@@ -55,15 +55,8 @@ services:
             - postgres
         command: mapswipe_workers --verbose run --schedule
         volumes:
-            - ./api-data:/var/lib/mapswipe_workers/api-data/:rw
-            - ./api-data/agg_results:/var/lib/mapswipe_workers/api-data/agg_results:rw
-            - ./api-data/groups:/var/lib/mapswipe_workers/api-data/groups:rw
-            - ./api-data/history:/var/lib/mapswipe_workers/api-data/history:rw
-            - ./api-data/projects:/var/lib/mapswipe_workers/api-data/projects:rw
-            - ./api-data/results:/var/lib/mapswipe_workers/api-data/results:rw
-            - ./api-data/tasks:/var/lib/mapswipe_workers/api-data/tasks:rw
-            - ./api-data/yes_maybe:/var/lib/mapswipe_workers/api-data/yes_maybe:rw
-            - ./api-data/hot_tm:/var/lib/mapswipe_workers/api-data/hot_tm:rw
+            - ./log:/var/lib/mapswipe_workers/:rw
+            - ./api-data:/home/mapswipe/.local/share/api-data/:rw
```

Comment:
I know that the lines above look really ugly. I think that I was struggling setting up the shared volumes between the `mapswipe_workers` container and the `api` container. Did you already double check that the api serves the data as we expect it?

Reply:
I will double check.

Reply:
I just run it on the dev server. At least projects/ and stats.csv has been written into the api-data dir by mapswipe_workers and successfully read by the api according to the timestamp.